### DISCUSSION TITLE
Welcome to MLT Discussions!

### DISCUSSION THREAD

**Discussion by ddennedy** [mltframework/mlt#1025] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by frdbr** [comment]
Greetings all, I am a video maker based in Brazil and member of the Kdenlive team. 

Cheers

**Comment by bmatherly** [comment]
Hello. I am a video expert based in the US. I am a developer for MLT and Shotcut.

**Comment by shadowfax65** [comment]
Hi, I am an IT project manager and hobby video maker in Canada. I am also a member of the Kdenlive core team.

**Comment by nuxttux** [comment]
Hi - Nuxttux here, I make video tutorial with & about Open Source programs, amongst other things. Happy to have found this.
